### ISSUE TITLE
Requesting a minor clarification in definition of Critical error

### ISSUE THREAD

**Issue by plannero** [w3c/silver#556]
In the draft WCAG3 published June 8th 2021, there's an example in the definition of "critical error" which could cause some confusion: 

It currently reads "example: submit button not in tab order". 
See https://www.w3.org/TR/2021/WD-wcag-3.0-20210608/#dfn-critical-error 

I believe there are at least three possible interpretations of that brief sentence: 
1. the button is not at all present in the sequence of tab stops - thus not at all reachable using keyboard only (definitely a critical error).
2. the button is present, but not in a logical position in the sequence of tab stops (unclear to me if this should be considered a critical error or not, it probably depends on HOW illogical the position is)
3. both of the two previous situations.

It is quite possible that the formulation is already unambiguous to a native speaker of English, but many WCAG readers, including myself, have English as a second language. 

Could somebody please propose a formulation that eliminates the risk of misunderstanding?

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
